app/templates/account/reset_password.html

```diff
@@ -6,6 +6,7 @@
         <div class="eight wide computer sixteen wide mobile centered column">
             <h2 class="ui dividing header">Reset your password</h2>
             {{ f.render_form(form) }}
+            {{ form.hidden_tag() }}
```

Comment:
doesn't this get called by the `begin_form` macro that happens when you say `f.render_form(form)`